debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/transforms/MongoDataConverter.java

```diff
@@ -243,14 +244,14 @@ public static void addFieldSchema(Entry<String, BsonValue> keyValuesforSchema, S
             break;
 
         case REGULAR_EXPRESSION:
-            SchemaBuilder regexwop = SchemaBuilder.struct();
+            SchemaBuilder regexwop = SchemaBuilder.struct().name(SCHEMA_NAME_REGEX);
```

Comment:
Will this work if there're multiple regex fields? I.e. shouldn't it be prefixed with the builder name as the others?

Reply:
If you look below the regex has fixed structure - so I thought it should be a static datatype like for example `Timestamp` is.

Reply:
Ok, makes sense.